shared/src/common/enums/exceptions/exception-message.enum.ts

```diff
@@ -17,6 +17,7 @@ enum ExceptionMessage {
   STORAGE_NOT_FOUND = 'Storage was not found',
   INVALID_USER_REQUEST = 'User request is invalid',
   INTERVIEW_EXIST = 'Interview for this category was either passed or is in progress',
+  INTERVIEW_DOES_NOT_EXIST = 'Interview with that ID does not exist',
```

Comment:
I've noticed that initially all exception messages have a `.` as ending but at some moment everyone forgot about it :c
Could you please add dots to in messages where they are missed?